source/07-processing/10-text-and-data-mining.rst

```diff
@@ -16,7 +16,7 @@ A key feature of TDM is the discovery of unknown associations based on categorie
 
 * COUNTER metrics do not record TDM itself, as most of this activity takes place after an article has been downloaded. Instead, metrics represent the count of articles downloaded for the purposes of mining.
 * Usage associated with TDM activity (e.g. articles downloaded for the purpose of TDM) MUST be tracked by assigning an Access_Method of TDM.
-* Usage associated with TDM activity MUST be reported using the Title, Database, and Platform Reports by identifying such usage as Access_Method=TDM.
+* Usage associated with TDM activity MUST be reported in the Title, Database, and Platform Reports by identifying such usage as Access_Method=TDM.
```

Comment:
Why only Title, Database, and Platform Reports? Access_Method TDM is also permissible for Item Reports, so I would expect this usage to also be reported in Item Reports.